src/GitVersion.Core/Git/ITag.cs

```diff
@@ -2,7 +2,7 @@ namespace GitVersion;
 
 public interface ITag : IEquatable<ITag?>, IComparable<ITag>, INamedReference
 {
-    string? TargetSha { get; }
+    string TargetSha { get; }
 
```

Comment:
Please remove the empty line to be consistent with other interfaces

Reply:
Okay no problem I change it. Just want to point out that it is hard to ready and maybe it is worth to think about to change this rule to increase the accessability to the code.

Reply:
Let's check this after we're done with the PR